CMake/cdat_modules/ipaddress_deps.cmake

```diff
@@ -0,0 +1 @@
+set(IPADDRESS_deps ${pip_pkg} ${python_pkg} ${setuptools_pkg})
```

Comment:
@doutriaux1 I'm not certain about the `pip_pkg` dependency, the `setup.py` contains:
```python
try:
    from setuptools import setup
except ImportError:
    from distutils.core import setup
```